web/src/router.rs

```diff
@@ -103,39 +125,95 @@ mod organization_endpoints_tests {
             let url = base.join(path.as_ref())?;
             Ok(url.as_str().to_string())
         }
+
+        /// Logs into a new AuthSession with a session cookie. The cookie will be
+        /// cached by the client Reqwest instance because we constructed the client
+        /// with it turned on (i.e. `cookie_store(true)`).
+        ///
+        /// This is meant to be reused by all tests that sit behind a protected route.
+        pub async fn login(&mut self, user: &user::Model) -> anyhow::Result<()> {
+            let creds = [("email", "test@domain.com"), ("password", "password2")];
+            let response = self
+                .client
+                .post(self.url("/login").unwrap())
+                .form(&creds)
+                .send()
+                .await?;
+
+            let response_text = response.text().await?;
+            debug!("response_text: {:?}", response_text);
+            assert_eq!(
+                response_text,
+                json!({
+                    "display_name": user.display_name,
+                    "email": user.email,
+                    "first_name": user.first_name,
+                    "last_name": user.last_name,
+                })
+                .to_string()
+            );
+
+            Ok(())
+        }
+
+        /// Creates a test user::Model entity instance that can be used by tests to
+        /// log in to the /login endpoint and create a valid AuthSession.
+        pub fn get_user() -> anyhow::Result<user::Model> {
```

Comment:
@calebbourg what do you think of this function name `get_user()`? Do you think it's confusing such that you'd expect some more dynamic getter of a property instead of generating a static instance of a `user::Model`?

Would `create_user()` or `create_test_user()` be better or do you think it's fine as is and I'm just overthinking it?